src/finufft.cpp

```diff
@@ -756,7 +759,9 @@ int FINUFFT_MAKEPLAN(int type, int dim, BIGINT *n_modes, int iflag, int ntrans,
       return FINUFFT_ERR_MAXNALLOC;
     }
 
-    p->fwBatch = FFTW_ALLOC_CPX(p->nf * p->batchSize); // the big workspace
+#ifndef FINUFFT_USE_DUCC0
+    timer.restart();
+    p->fwBatch = (CPX *)FFTW_ALLOC_CPX(p->nf * p->batchSize); // the big workspace
```

Comment:
This cast is guaranteed to work correctly, since C++ guarantees the memory layout of std::complex to be compatible to the FFTW type.